libc/test/src/__support/FPUtil/fpbits_test.cpp

```diff
@@ -427,11 +427,9 @@ TEST(LlvmLibcFPBitsTest, DoubleType) {
 
 #ifdef LIBC_TARGET_ARCH_IS_X86
 TEST(LlvmLibcFPBitsTest, X86LongDoubleType) {
+#ifndef LIBC_TYPES_LONG_DOUBLE_IS_FLOAT64
```

Comment:
Can you replace the above condition `LIBC_TARGET_ARCH_IS_X86` with `LIBC_TYPES_LONG_DOUBLE_IS_X86_FLOAT80` instead.

Reply:
Updated.